.babelrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
Oh, I completely forgot that customizing the babelrc file in CRA is, well, difficult.  Does this actually work and not break the project?

Reply:
Storybook runs. All the existing tests pass as well. However, the project itself breaks due to compilation errors caused by QuestionQueue.stories.tsx. 

Error:  Type 'EventEmitter' is missing the following properties from type 'Socket': io, nsp, id, connected, and 10 more.

I don't think this was caused by the installation and set up of the plug-in